### ISSUE TITLE
FR: Allow for exporting a excalidraw with embedded notes

### ISSUE THREAD

**Issue by Watenk** [zsviczian/obsidian-excalidraw-plugin#2069]
### Is your feature request related to a problem? Please describe.

I wanted to share a excalidraw but when I exported it to a PNG all the embedded notes only contained their name (like in #1449). 

### Describe the solution you'd like

A method to export a excalidraw with embedded notes included.

### Real-life usage scenarios

So you can share your excalidraw's with embedded notes in them and save them in PNG format.

### Describe alternatives you've considered

You can embed notes as images but then images inside the notes won't show.

### Additional context

![image](https://github.com/user-attachments/assets/42592929-d817-4bf7-a2ca-e71a789cb59c)
For example, this is an excalidraw I want to export but it looks like this after exporting. 
![Linux](https://github.com/user-attachments/assets/978bd3eb-57bf-492c-90d9-887a9346239e)
And its not convenient to take screenshots of parts and then stitch them together.

**Comment by zsviczian** [comment]
yes, this has been on my mind for some time. Note that that you can embed markdown as image. It has many limitations... but it is worth testing, as that is how it will/would look when/if I implement this feature. It is not implemented because it is very resource intensive.

**Comment by Watenk** [comment]
The only problem with embedding notes as images is that images inside notes won't show. Otherwise its fine.

![image](https://github.com/user-attachments/assets/fb8381bc-1036-4a59-81e1-098dc70bf67f)

**Comment by zsviczian** [comment]
yes PNGs won't show, that is because within an <img> element the local obsidian://blob is not accessible. If you encapsulate your images into Excalidraw drawings, i.e. you place the line example.png into an Excalidraw and embed that into your note, then the image will show.

**Comment by mtuckerb** [comment]
I would like to +1 this request. The only way I can see to have truly hybrid notes, is to use back of the note cards (embeds) and type into them. Constantly switching between Obsidian and Excalidraw is a non starter, and not being able to export make it all pretty uninteresting.

**Comment by jiajiala** [comment]
+1

**Comment by AnasBerka** [comment]
+1

**Comment by jorge-jrzz** [comment]
+1

**Comment by tuntunhuang** [comment]
+1

**Comment by zsviczian** [comment]
Already available. Use the printscreen feature in the export dialog.

**Comment by tuntunhuang** [comment]
> Already available. Use the printscreen feature in the export dialog.
> […](#)

Thank u~ 
I updated my excalidraw and tried printscreen feature. It's amazing, thank u again~

**Comment by MurrayGroves** [comment]
Would love if there was a way to do this with the auto-export features 🙏
